src/inmanta/server/services/compilerservice.py

```diff
@@ -448,8 +449,15 @@ async def setup() -> AsyncIterator[Awaitable[Optional[data.Report]]]:
 
             server_address = opt.server_address.get()
             server_port = opt.server_bind_port.get()
-            cmd = [
-                "-vvv",
+
+            app_cli_args = ["-vvv"]
+            # Pass down the logging config to the compiler
+            compiler_log_config_file: str | None = compiler_log_config.get()
+            if compiler_log_config_file:
+                app_cli_args.append("--logging-config")
+                app_cli_args.append(os.path.abspath(compiler_log_config_file))
```

Comment:
I wonder if it would not be better to pass along the config file and folder of the server in full? 

i.e. the config folder defaults to `/etc/inmanta/inmanta.d` so if we don't pass it along, we will read `/etc/inmanta/inmanta.d` wether the server uses it or not.